routers/web/auth/oauth.go

```diff
@@ -892,8 +895,11 @@ func SignInOAuthCallback(ctx *context.Context) {
 		return
 	}
 
+	fmt.Println("hier hier")
```

Comment:
![here](https://media.giphy.com/media/l2Jei24C2J7dsKhby/giphy.gif)

(couple more print statements down below)

Reply:
ups, removed the debug prints...